src/server/services/wcs/qgswcs.cpp

```diff
@@ -74,7 +74,7 @@ namespace QgsWcs
         if ( req.isEmpty() )
         {
           throw QgsServiceException( QStringLiteral( "OperationNotSupported" ),
```

Comment:
Is `QgsServiceException::OGC_OPERATION_NOT_SUPPORTED` available in this context and the other places ?

Reply:
nop, only in WMS context.

Reply:
For now, I only add an enum for OGC exceptions within the WMS service/module. So it's not available in WCS / WMTS / ... yet.

Reply:
Thanks for the explanation.